src/tokens.rs

```diff
@@ -276,22 +276,44 @@ pub fn process_distribute_tokens<T: Client>(
     client: &ThinClient<T>,
     args: &DistributeTokensArgs<Box<dyn Signer>>,
 ) -> Result<(), Error> {
-    let mut rdr = ReaderBuilder::new()
-        .trim(Trim::All)
-        .from_path(&args.bids_csv)?;
-    let bids: Vec<Bid> = rdr.deserialize().map(|bid| bid.unwrap()).collect();
-    let mut allocations: Vec<Allocation> = bids
-        .into_iter()
-        .map(|bid| create_allocation(&bid, args.dollars_per_sol))
-        .collect();
+    let input_csv = if let Some(i) = &args.allocations_csv {
```

Comment:
@garious I could use a little guidance here.  Either args.allocations_csv or args.bids_csv will have Some value, as required by the way the clap args are set, but this won't compile because the `else` clause is not exhaustive if args.bids_csv is None, though we should never actually reach that block if that is the case.  What is the Rust way to do this?

Reply:
I ran into this issue last night, which is partly why I originally settled on setting --allocations_csv as a bool rather than Option<String>, but I'm probably just missing some programmatic way to approach this

Reply:
Okay, let's move forward with `input_csv`, but can you change `--allocations_csv:bool` to `--from-bids:bool`?  That is, make the auction input the exception, not the norm.

Regarding your question, yeah, learn more Rust please.  `match (allocations_csv, bids_csv)` would have done the trick.